super_csv/serializers.py

```diff
@@ -0,0 +1,28 @@
+"""
+Serializers for CSV operation data.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
```

Comment:
I missed this before. To make this library more reusable, this should be `from django.contrib.auth import get_user_model`, like it is in models.py